includes/Modules/Sign_In_With_Google.php

```diff
@@ -53,52 +66,208 @@ final class Sign_In_With_Google extends Module implements Module_With_Assets, Mo
 	public function register() {
 		add_filter( 'wp_login_errors', array( $this, 'handle_google_auth_errors' ) );
 
-		add_action( 'login_form_google_auth', array( $this, 'handle_google_auth' ) );
+		add_action( 'login_form_google_auth', $this->get_method_proxy( 'handle_auth_callback' ) );
```

Comment:
Is it something that we need to follow now? Frankly, i am not a fun of neither method proxy nor arrow functions. Why not to make action/filter callbacks be public functions?

Reply:
That's a fair point. It was definitely a convention early on in the project to avoid or at least be more intentional about introducing public methods. I'm not against it, but I do think we want to be mindful about the public interface.